app/assets/javascripts/components/story/task/TasksList.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Task from './Task'
+import PropTypes from 'prop-types';
+
+const TasksList = ({ tasks, onDelete, onUpdate }) => (
+  <div>
+    {
+      tasks.map((task) => (
+        <Task
+          key={task.id}
+          task={task}
+          onDelete={() => onDelete(task.id)}
+          onUpdate={() => onUpdate(task, { done: !task.done })}
```

Comment:
I think it would be better to call it as `onToggle` because the only thing we can do to update a task is toggling its status right?

Reply:
yes! Changed!